llvm/lib/Target/CSKY/CSKYISelDAGToDAG.cpp

```diff
@@ -116,8 +116,7 @@ void CSKYDAGToDAGISel::Select(SDNode *N) {
 
 bool CSKYDAGToDAGISel::selectInlineAsm(SDNode *N) {
   std::vector<SDValue> AsmNodeOperands;
-  unsigned Flag;
-  InlineAsm::Kind Kind;
+  InlineAsm::Flag Flag(0);
```

Comment:
Maybe instead of using `0` here you could have a default c'tor that will zero out the fields instead. The `0` just looks a bit weird here.

Reply:
done in https://github.com/llvm/llvm-project/pull/65649/commits/81cec479e37487489b00bed2eff3502e754692df